github/libp2p/membership.json

```diff
@@ -107,48 +65,21 @@
   "dirkmc": {
     "role": "member"
   },
-  "dryajov": {
-    "role": "member"
-  },
   "dvdplm": {
```

Comment:
Same here - October, e.g. https://github.com/libp2p/rust-libp2p/pull/2085#issuecomment-946495027

Reply:
I don't think @dvdplm requires access to the libp2p org. Contributions are very much appreciated, though they don't require special access. Please comment here in case I am missing something @dvdplm.

Reply:
You are not missing anything, as usual. :)
I can be removed.